### DISCUSSION TITLE
Marking all posts that *don’t* contain “http” as read

### DISCUSSION THREAD

**Discussion by mikae1** [FreshRSS/FreshRSS#6087] [Q&A] [answered]
I just can’t wrap my head around creating user queries. 

For a certain feed I want to mark all posts that _don’t_ contain “http” as read. I have succeeded creating something similar for another feed, where it marks every post that contains “twitter” as read, but I just can’t figure out how to do it again. 

In user queries it looks like this:
![image](https://github.com/FreshRSS/FreshRSS/assets/773707/8dbaee49-7390-451a-a238-69e82b22eea4)

Again, how would I go about marking posts in a certain feed as read if they _don’t_ contain “http”?

**Comment by Alkarex** [comment, marked as answer]
Marking as read automatically is done via *Filter actions* (not by user queries) under a feed's settings:

![image](https://github.com/FreshRSS/FreshRSS/assets/1008324/512b737f-e31c-4728-84c5-d9467c4f615b)

For matching feeds, which do not contain a keyword, you can use a negation such as `!http`. Watch out that precisely `http` is something that is probably in most articles, as it is used in links for instance, so you might have to be a bit more precise.

**Reply by mikae1** [reply]
Thank you!
